KarhooUISDK/Screens/BookingScreen/KarhooBookingViewController.swift

```diff
@@ -223,6 +250,31 @@ final class KarhooBookingViewController: UIViewController, BookingView {
         showAsOverlay(item: alertController, animated: true)
     }
 
+    private func setCoverageView(_ hasCoverage: Bool?) {
+        func setCoverageView(isVisible: Bool) {
+            let targetAlpha: CGFloat = isVisible ? 1 : 0
```

Comment:
We could use the alpha constants we have, 1 for enabled and 0 for hidden

Reply:
Updated to use UI Constants, but for the future I think we should avoid using such external references to 1/0 alpha values. This is because those values have clear graphic engine outcome – the view is intractable only if alpha equals 1, and it is opaque only if the value is 0. In every other case (like shadow, inactive etc.) storing a common value makes a lot of sense.